rest/sgcollect.go

```diff
@@ -37,8 +42,11 @@ const (
 )
 
 type sgCollect struct {
-	cancel context.CancelFunc
-	status *uint32
+	cancel        context.CancelFunc
+	status        *uint32
+	sgPath        string
+	sgCollectPath string
+	err           error
```

Comment:
Call this something like `pathErr` to make the usage clear.

```suggestion
	pathErr       error
```

Reply:
This is changed to pathError.